### PR TITLE
Contribution guidelines

### LINKED ISSUE #175: Create contribution guidelines for project
- [ ] Add CODEOWNERS file
- [ ] Add code of conduct ([Contributor Covenant](https://www.contributor-covenant.org/version/1/4/code-of-conduct))
- [ ] Remove custom issue/pull request templates
- [ ] Add code style guidelines
- [ ] Add test coverage guidelines
- [ ] Add performance guidelines
- [x] Add supported browsers list (probably add this to creativecommons.github.io and link here)

[Github instructions for adding contributor guidelines](https://help.github.com/articles/setting-guidelines-for-repository-contributors/)

See: https://wikijs.creativecommons.org/tech/github-repo-guidelines (only accessible to CC staff)

### PR DESCRIPTION
**Pull request by brenoferreira**
Improvements to our contribution guidelines documentation

Fixes #175

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -12,7 +12,68 @@ Please consult the [README](./README.md) file at the root of this repository.
...
+
+### Bugs
+
+If you find a bug, please open an issue in this repository describing the bug. The bug description should contain:
+
+* Step-by-step to reproduce
+* What the expected behaviour is
+* Which browsers, including their versions, and operating system were used to reproduce the bug
+
+Example of a good bug description:
```
Comment (kgodey) [inline CONTRIBUTING.md]: This is covered in our bug template that we already have set up when filing an issue on GitHub and these instructions are a little bit different from the template.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -12,7 +12,68 @@ Please consult the [README](./README.md) file at the root of this repository.
...
+
+If you have an idea of a new feature or change to how the CC Search frontend works, please file an issue so we can discuss the possibility of that change or new feature being implemented and released in the future. This lets us come to an agreement about the proposed idea before any work is done.
+
+### Pull requests
+
+Before you start writing code, make sure there is an issue open. Pull requests without a link to an existing issue won't be merged.
+
+If you want to get started contributing code to this project but don't know exactly what to work on, we compiled a good list of issues labeled as [Good first issues](https://github.com/creativecommons/cccatalog-frontend/labels/good%20first%20issue) which are small in scope and not so complex to solve. There is also issues labeld as [Help wanted](https://github.com/creativecommons/cccatalog-frontend/labels/help%20wanted) which can be a bit more complex but are good examples of thigns we are currently accepting help from the community.
+
+Once you open your Pull Request, make sure that its description contains the following information:
```
Comment (kgodey) [inline CONTRIBUTING.md]: I think this should go in the PR template.

Comment (brenoferreira) [reply CONTRIBUTING.md]: In the #175 issue description it says "Remove custom issue/pull request templates". Are we keeping them?

Comment (kgodey) [reply CONTRIBUTING.md]: No, we should remove them here, they will be automatically pulled in from the ones in https://github.com/creativecommons/.github.

Comment (kgodey) [reply CONTRIBUTING.md]: See https://wikijs.creativecommons.org/tech/github-repo-guidelines (only accessible to CC staff) for more details.

**Comment by kgodey** [comment]
feedback from our call: please modify PR template to use DCO.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -12,7 +12,33 @@ Please consult the [README](./README.md) file at the root of this repository.
 
 ## How to Contribute
 
-Please follow the processes in our general [Contributing Code](https://creativecommons.github.io/contributing-code/) guidelines on the Creative Common Open Source website.
+Please read the processes in our general [Contributing Code](https://creativecommons.github.io/contributing-code/) guidelines on the Creative Common Open Source website. It contains some general instructions that should be followed when contributing to any of the Creative Commons open-source repositories.
```
Comment (kgodey) [inline CONTRIBUTING.md]: I think it would be worth mentioning here that `CODEBASE.md` is a good place to find high-level information.

**Pull request merged**